_overviews/scala3-book/scala-tools.md

```diff
@@ -284,11 +313,18 @@ As with the previous lesson, create an sbt project directory structure for a pro
 $ mkdir HelloScalaTest
 $ cd HelloScalaTest
 $ mkdir -p src/{main,test}/scala
-$ mkdir project target
+$ mkdir project
 ```
 
 
-### Creating the build.sbt file
+### Creating the build.properties and build.sbt files
```

Comment:
I think I may need to look at this file when my brain is a little fresher, but at the moment it seems like I created a lot of `###` headers under the `## Building Scala projects with sbt` header, and today I find that confusing. I have another project I need to work on now, but if you’d like, I can look at this file again next week and see if it makes sense then, or reorganize it a little.

Reply:
Feel free to reorganize if you think it makes sense :blush: 

IMHO, the structure looks good:
![Screenshot from 2021-06-23 09-09-02](https://user-images.githubusercontent.com/332812/123053093-d47ec400-d403-11eb-925b-a2eefd139a00.png)

Reply:
Okay, thanks. I just created a reminder to look at it next Wednesday.